docs/accordproject-faq.md

```diff
@@ -0,0 +1,56 @@
+
+Smart Legal Contracts can be easily searched, analyzed, queried, and understood. By associating a data model to a contract, it is possible to extract a host of valuable data about a contract or draft a series of contracts from existing data points (i.e., variables and their values).
+
+The data model is used to ensure that all of the necessary data is present in the contract, and that this data is valid. In addition, it provides the necessary structure to enable contracts to "come alive" by adding executable logic.
+
+The result is a contract that is:
+
+
+* Searchable —
+* Analyzable —
+* Real-time —
+* Integrated —
+
+Consequently, contracts are transformed from business liabilities in constant need of management to assets capable of providing real business intelligence and value.
+
+### What is the Accord Project and what is its purpose?
+
+The Accord Project is a non-profit, member-driven organization that builds open source code and documentation for smart legal contracts for use by transactional attorneys, business and finance professionals, and other contract users. Open source means that anyone can use and contribute to the code and documentation and use it in their own software applications and systems free of charge.
+
+The purpose of the Accord Project is to establish and maintain a common and consistent legal and technical foundation for smart legal contracts. The Accord Project is organized into working groups focused on transactions: supply chain, financial services, intellectual property, venture and token sales, real estate and construction; and also on dispute resolution. The transactional working groups are assisted by the Technology Working Group, which builds the underlying open source code and specifications to codify the knowledge of the transactional working groups. More details about the internal governance of the Accord Project are available here.
+
+
+### How can I get involved?
+
+The working groups have calls at least once a month and use the Accord Project’s Slack group chat application (join by clicking here) for discussion. The dates, dial-in instructions, and agendas for the working groups are all listed in the Project’s public calendar and typically also in working group’s respective slack channels.
+
+A primary purpose of the working groups is to develop a universally accessible and widely used open source library of modular, smart legal contract and smart clause templates and models that reflect input from transactional attorneys and other experts. Smart legal contract templates are built according to the Project’s Cicero Specification.
```

Comment:
Could provide a link for `... Accord Project’s Cicero Specification`

Reply:
Remove `smart clause1. Simply `smart templates`?

Reply:
done